components/client/src/main/java/com/hotels/styx/client/OriginRestrictionLoadBalancingStrategy.java

```diff
@@ -48,44 +48,44 @@
     private final LoadBalancingStrategy delegate;
     private final String cookieName;
 
-    OriginRestrictionLoadBalancingStrategy(ActiveOrigins activeOrigins, LoadBalancingStrategy delegate, String cookieName) {
+    public OriginRestrictionLoadBalancingStrategy(ActiveOrigins activeOrigins, LoadBalancingStrategy delegate, String cookieName) {
         this.activeOrigins = activeOrigins;
```

Comment:
`activeOrigins` field should be final.
`activeOrigins` parameter should be null-checked.